bin/blocks-continue

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-import logging
-from argparse import ArgumentParser
-
-from blocks.scripts import continue_training
-
-if __name__ == "__main__":
-    logging.basicConfig()
-
-    parser = ArgumentParser("Continues your pickled main loop")
-    parser.add_argument(
-        "path", help="A path to a file with a pickled main loop")
-    args = parser.parse_args()
-
-    continue_training(args.path)
```

Comment:
We could keep it, but the main issue is that pickle doesn't play nice with functions or objects defined in the `__main__` namespace of the training script. If a user relied on those, they won't be able to use `blocks-continue`. That's why @dmitriy-serdyuk decided it was probably more robust to ask the user to rely on `continue_training`, with better documentation that explains how to use `continue_training` from within the training script (where the global namespace is the same).